pkgs/development/embedded/fpga/ipbus-uhal/uhal.nix

```
@@ -0,0 +1,72 @@
+{
+  lib,
+  stdenv,
+  bash,
+  cacert,
+  boost,
+  pugixml,
+  python3,
+  python3Packages,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "uhal";
+  version = "2.8.16";
+  src = fetchFromGitHub {
+    owner = "ipbus";
+    repo = "ipbus-software";
+    hash = "sha256-R+a9VmONyWh3BEYoMjRcXKv+3HaNcKbJDnYH1hXHdPg=";
+    rev = "refs/tags/v${version}";
+  };
+
+  patches = [
+    ./avoid-bin-bash.patch
+  ];
+
+  nativeBuildInputs = [
+    cacert
+    python3
+    python3.pkgs.distutils
+  ];
+  buildInputs = [
+    boost
+    pugixml
+    python3.pkgs.distutils
+    python3.pkgs.pybind11
+  ];
+  py3 = python3;
+  configurePhase = ''
+    echo "PATCHING SHEBANGS"
+    patchShebangs  uhal/config/install.sh
+    patchShebangs uhal/tests/setup.sh
+    patchShebangs scripts/doxygen/api_uhal.sh
+    patchShebangs config/progress.sh
+    patchShebangs config/Makefile.macros
+  '';
```

You probably can get away with just running patchShebangs recursively on every file. It won't hurt

```suggestion
  preConfigure = ''
    patchShebangs .
  '';
```

---

Is it ok to leave it like this so as not to break something.